sslSniffer/sslSnifferTest/snifftest.c

```diff
@@ -71,6 +78,8 @@ int main(void)
     #include <sys/socket.h>    /* AF_INET */
     #include <arpa/inet.h>
     #include <netinet/in.h>
+#else
+    #include <stdint.h>        /* uint8_t */
```

Comment:
Please use `byte` and remove include of stdint.h.

Reply:
Done